src/main/java/net/dv8tion/jda/internal/interactions/CommandInteractionImpl.java

```diff
@@ -123,6 +125,19 @@ private void parseResolved(JDAImpl jda, DataObject resolveJson)
         }
     }
 
+    private void buildText()
+    {
+        //Get text like the text that appears when you hover over the interaction in discord
+        StringBuilder builder = new StringBuilder();
+        builder.append("/").append(getCommandPath().replace("/", " ")).append(" ");
+        //build options (formatted appropriately)
+        for (OptionMapping o : this.options)
+        {
+            builder.append(o.getName()).append(":").append(o.getAsString()).append(" ");
```

Comment:
This will not be correct for a number of option types. For instance, a channel will return the id here rather than `#name`.

Reply:
added some checks for mentionables